damnit/backend/db.py

```diff
@@ -185,50 +129,25 @@ def kafka_topic(self):
     def path(self):
         return self._path
 
+    def _set_schema_version(self, version: int):
+        self.metameta["data_format_version"] = int(version)
```

Comment:
Is the explicit conversion necessary since the type hint says `int`? If so then the type hint should probably be changed.
```suggestion
        self.metameta["data_format_version"] = version
```